core/src/main/java/tc/oc/pgm/integration/VanishIntegrationImpl.java

```diff
@@ -0,0 +1,18 @@
+package tc.oc.pgm.integration;
+
+import org.bukkit.entity.Player;
+import tc.oc.pgm.api.integration.VanishIntegration;
+import tc.oc.pgm.api.player.MatchPlayer;
+
+public class VanishIntegrationImpl implements VanishIntegration {
```

Comment:
this needs a better than no-op default, even if community plugin isn't used vanish shouldn't just *not work*